lib/RESTClient.js

```diff
@@ -23,6 +24,23 @@ function errorMap(mdError) {
     return map[mdError] ? map[mdError] : mdError;
 }
 
+/**
+ * Turns the bootstrap list into an array of host/port objects.
+ *
+ * @param {String[]} list - the list to process
+ *
+ * @return {{ host: string, port: number }[]}
+ */
+function parseBootstrapList(list) {
+    return list.map(elem => {
+        const tmp = elem.split(':');
+        const host = tmp[0];
+        const port = Number.parseInt(tmp[1], 10);
```

Comment:
So we're assuming that the port is **_always_**  specified? Should we not make this optional, and default to the default port (9000) ?